docs/osf-trivia-list/dasharo.md

```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
+    - [trustworthiness for all](#dasharo-trustworthiness-for-all).
+
+### Dasharo Zero-Touch Initial Deployment
+
+Documentation supported hardware provide information about initial deployment,
+updates and recovery procedures. Developed by Dasharo Team [Dasharo Tools Suite
+(DTS)][dtsos] operating system support users by automating the deployment
+process, which help reducing errors and inconsistencies, and make sure the
+firmware can be further updated to new version without any problems. DTS also
+provide controlled and secure environment for initial deployment and update of
+firmware, reducing the risk of tampering or unauthorized changes.
+
+Dasharo Zero-Touch Initial Deployment is smooth, effortless and user-friendly
+process, which reduce user frustration and improve satisfaction.
+
+Use of DTS largely improve firmware adoption, hardware compatibility reporting
+and binary blobs transmission, as well as recovery.
+
+For more details about zero-touch initial deployment please read relevant
+Dasharo Tools Suite
+[documentation](../../dasharo-tools-suite/documentation/#dasharo-zero-touch-initial-deployment).
+
+### Dasharo Clean and Simple Code
+
+* Each supported platform has its own branches in Dasharo's coreboot repository:
+    - `<board_name>/develop`, e.g. `msi_ms7d25/develop`, used for bug fixes and
+    development of new features.
+    - `<board_name>/release`, e.g. `msi_ms7d25/release`, contains only the
+    released and validated code. It is used to build the firmware binaries
+    distributed to the community through Dasharo releases.
+* Each supported platform has a build scripts included in Dasharo's coreboot
+  repository to simplify the complex build instructions
+
+### Dasharo Long Term Maintenance
+
+* We provide long term maintenance - coreboot community for various reasons, do
+  not merge some patches, because of understaffing, lack of reviewers. Some
+  changes have long way to upstream, we maintain those patches and make them
+  work before those will go upstrea. If ever, we are committed to maintain
+  platforms which are moved to branch in coreboot.
+* Firmware update - we are registered [consultants for fwupd/LVFS][lvfs] and
+  enable customers and community platforms, so they can get seamless firmware
+  update in Linux.
+
+### Dasharo Support Package
+
+* Support - Dasharo Team is paid to provide support in the community through
+  [Dasharo Matrix Space][dasharo-matrix], of course we want to introduce some
+  commercial products related to the support to make the business feasible, but
+  some level of support always would be available to the community.
+
+### Dasharo Transparent Validation
+
+* We provide transparent validation results - coreboot in itself provide no
+  guarantees around release quality and do not provide binary distribution (for
+  reference please check [coreboot project scope][coreboot-scope], we provide
+  those in scope of validation we perform.
+
+### Dasharo Trustworthiness for All
```

Comment:
We define the same term here: https://github.com/Dasharo/docs/pull/507/files#diff-c8be10a4380efbce4c82013c5d1e8d7d42423eda7107a96990f0c54284767b2cR307
But in different manner

Reply:
Fixed: https://github.com/Dasharo/docs/commit/a51ca4d68bd9f339f71f23e3edc75f34c21f3018